platform/working-with-vsp/orientation/request-access-to-tools.md

```diff
@@ -17,7 +17,7 @@ To work on the Veteran-facing Services Platform, each team member needs to use t
 
 ## GitHub
 
-1. If you haven't already, check your email for an invitation to the Department of Veterans Affairs GitHub Organization, and join! If you have not received an invitation please follow the steps in [Github User Requests](https://github.com/department-of-veterans-affairs/github-user-requests).
+1. If you haven't already, check your email for an invitation to the Department of Veterans Affairs GitHub Organization, and join! If you have not received an invitation (and do not already have a va.gov email), please reach out to your PM and/or team lead to request they submit a request for github access. If you already possess a va.gov email and have not received an invitation, please follow the steps in [Github User Requests](https://github.com/department-of-veterans-affairs/github-user-requests).
```

Comment:
the second part won't work - if you aren't part of the org, you cxan't see github-user-requests - so we probablyl need to copy a brief version of those instructions over, mainly the point about the VA-internal link you cxan use.

Reply:
looks good otherwise

Reply:
ah yeah, that's totally fair

Reply:
added in https://github.com/department-of-veterans-affairs/va.gov-team/pull/11950/commits/31b34f9f534880b6b28afc416bdbf01b9119675c

Reply:
improved up in https://github.com/department-of-veterans-affairs/va.gov-team/pull/11950/commits/6e8b9e0ae00fddf05e31fec5b65c5013c6a72b26